src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,664 @@
+    return [
+      {
+        key: 'people-picker-no-results-item',
+        content: (
+          <Text weight="bold" content={noResultsMessage || `We couldn't find any matches.`} />
+        ),
+        styles: {
+          backgroundColor: variables.listItemBackgroundColor,
+        },
+      },
+    ]
+  }
+
+  private renderActiveValues(styles: ComponentSlotStylesInput) {
+    const value = this.state.value as DropdownListItem[]
+    const { getA11yRemoveItemMessage } = this.props
+
+    if (value.length === 0) {
+      return null
+    }
+
+    return value.map(item => {
+      const optionalImage = {
+        ...(item.image && {
+          image: { src: item.image, avatar: true, onClick: this.handleLabelClick },
+        }),
+      }
+      return (
+        <Label
+          role="presentation"
+          styles={styles.activeListLabel}
+          circular
+          key={`active-item-${item.key || item.header}`}
+          content={<Text content={item.header} onClick={this.handleLabelClick} />}
+          {...optionalImage}
+          icon={{
+            name: 'close',
+            onClick: this.handleCloseIconClick.bind(this, item),
+            onKeyDown: this.handleCloseIconKeyDown.bind(this, item),
+            'aria-label': getA11yRemoveItemMessage
+              ? getA11yRemoveItemMessage(item)
+              : `Remove ${item.header} from selection.`,
+            'aria-hidden': false,
+            role: 'button',
+          }}
+        />
+      )
+    })
+  }
+
+  RefProvidingWrapper = ({ children }) => (
+    <Ref innerRef={domNode => _.invoke(children.props, 'innerRef', domNode)}>{children}</Ref>
+  )
+
+  private stateReducer = (
+    state: DownshiftState<DropdownListItem>,
+    changes: StateChangeOptions<DropdownListItem>,
+  ) => {
+    switch (changes.type) {
+      case Downshift.stateChangeTypes.changeInput:
+        this.trySetState({
+          searchQuery: changes.inputValue,
+        })
+        this.setState({
+          backspaceDelete: !(state.inputValue.length > 0 && changes.inputValue.length === 0),
+        })
+        _.invoke(this.props, 'onSearchQueryChange', changes.inputValue)
+        return changes
+      default:
+        return changes
+    }
+  }
+
+  private getA11yStatusMessage = ({
+    highlightedItem,
+    isOpen,
+    itemToString,
+    previousResultCount,
+    resultCount,
+    selectedItem,
+  }: DownshiftA11yStatusMessageOptions<DropdownListItem>) => {
+    if (!isOpen) {
+      return selectedItem ? itemToString(selectedItem) : ''
+    }
+    if (!resultCount) {
+      return 'No results are available.'
+    }
+    if (!highlightedItem && resultCount !== previousResultCount) {
+      return `${resultCount} result${
+        resultCount === 1 ? 'is' : 's are'
+      } available, use up and down arrow keys to navigate. Press Enter key to select.`
+    }
+    return ''
+  }
+
+  private setA11yStatus = (statusMessage: string) => {
+    const elementId = 'stardust-dropdown-a11y-status'
```

Comment:
I did not, to be honest, but I was suggested to use Portal at some point. I could not, as I had to conditionally render its content.
I had to start with a false condition var on state, then change it to true on componentHasMounted. But then I ended up with a <div> for each Dropdown on the page. When they got rendered, no Portal knew there are others rendering as well. And we only need one. So ... here we are.

Reply:
I'm reopening this so that others are aware that we're introducing this code. 

fyi @kuzhelov @mnajdova @miroslavstastny 

I don't think it's a good idea to have this approach but also don't want to block you.

@silviuavram Since it doesn't seem critical functionality, what if we remove this feature for now (`setA11yStatus`) and add it as a TODO to #560 until we find a better way to create this message?